src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+            },
+            {
+                begin: '\\b(?:FALSE|TRUE)\\b'
+            },
+            {
+                begin: '\\b(?:ZERO_ROTATION)\\b'
+            },
+            {
+                begin: '\\b(?:EOF|JSON_(?:ARRAY|DELETE|FALSE|INVALID|NULL|NUMBER|OBJECT|STRING|TRUE)|NULL_KEY|TEXTURE_(?:BLANK|DEFAULT|MEDIA|PLYWOOD|TRANSPARENT)|URL_REQUEST_(?:GRANTED|DENIED))\\b'
+            },
+            {
+                begin: '\\b(?:ZERO_VECTOR|TOUCH_INVALID_(?:TEXCOORD|VECTOR))\\b'
+            }
+        ]
+    };
+
+    var LSL_FUNCTIONS = {
+        className: 'function',
```

Comment:
The 'function' class name is for user-defined functions (like `function something()` in JS, for example), and these look like built-ins.